.github/workflows/docs-ci.yaml

```diff
@@ -28,15 +34,24 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash -l {0}
```

Comment:
Doesn't the actual `make …` step also need Conda activated (when Conda is in use)?

Reply:
Of course 🤦 I'll fix that. Also created #32 to catch this in CI.

Reply:
Fixed with [force-push](https://github.com/nextstrain/.github/compare/f1a8182ae488bc0aaf8e31e7f08c7b24e06498cd..79386fbfe9bf8702f54edb988e0f907fc6806697).